src/utils/payment-providers-mapper.ts

```diff
@@ -1,10 +1,27 @@
-export default function (provider) {
+import { Option } from "../types/shared"
+
+export default function (provider: string): Option {
```

Comment:
suggestion: I suggest the following simplification which could also allow to extract the providers elsewhere if needed and also simplify to add/remove or update the providers. wdyt?
```ts
const providerDescriptors: { [providerKey: string]: Option } = {
  stripe: {
    label: "Stripe",
    value: "stripe",
  },
  "stripe-ideal": {
    label: "Stripe Ideal",
    value: "stripe-ideal",
  },
  //...
}

export default function (provider: string): Option {
  return providerDescriptors[provider] ?? {
    label: provider,
    value: provider,
  }
}
```

Reply:
it would make it easily configurable and avoid having to update the core function each time we want to update the providers